Formula/dotnet.rb

```diff
@@ -92,11 +90,6 @@ def install
 
     (buildpath/"src/SourceBuild/tarball/patches/msbuild").install resource("homebrew-msbuild-patch")
 
-    # Fix usage of GNU-specific flag.
-    # TODO: Remove this when upstreamed
-    inreplace "src/SourceBuild/tarball/content/repos/Directory.Build.targets",
-              "--block-size=1M", "-m"
-
     Dir.mktmpdir do |sourcedir|
       system "./build.sh", "/p:ArcadeBuildTarball=true", "/p:TarballDir=#{sourcedir}"
```

Comment:
From local attempt, ARM build is failing as it is trying to get x64 runtime.

Reply:
ARM build breakage is probably due to https://github.com/dotnet/install-scripts/pull/314. Manually downloading previous copy of script no longer tries to download x64 tarballs.